### PR TITLE
Paid cards in editorial containers

### PR DESCRIPTION
**Pull request by lps88**
Add the correct card classes related to sizing at different breakpoints

## Screenshots
Before:
image

After:
image

With-logo variant:
image

## Tested in CODE?
Not Yet

### CODE REVIEW

File: common/app/layout/Sublink.scala [hunk 1]
```diff
@@ -352,12 +352,15 @@ case class PaidCard(
...
   fallbackImageUrl: Option[String],
   targetUrl: String,
+  cardTypes: Option[ItemClasses] = None,
   branding: Option[Branding]
 ) extends FaciaCard
 
 object PaidCard {
 
-  def fromPressedContent(content: PressedContent): PaidCard = {
+  def fromPressedContent(content: PressedContent): PaidCard = this.fromPressedContent(content, None)
```
Comment (TBonnin) [inline common/app/layout/Sublink.scala]: No need for a second functions: You can simply declare a second argument with a default value.

**Review by TBonnin — CHANGES_REQUESTED** [review]

**Comment by ScottPainterGNM** [comment]
Looks great from the visuals side, probably need someone more knowledgable for the code. Kelvin is off this week so maybe @rich-nguyen can help?

File: common/app/layout/Sublink.scala (hunk 1 above)
Comment (lps88) [inline common/app/layout/Sublink.scala]: Yeah initially wrote it that way but it gave me compilation errors:

```
[error] /Users/lydia_shepherd/dev/frontend/common/app/common/commercial/ContainerModel.scala:32: type mismatch;
[error]  found   : (model.pressed.PressedContent, Option[layout.ItemClasses]) => layout.PaidCard
[error]  required: model.pressed.PressedContent => ?
[error]     val cards = collection.curatedPlusBackfillDeduplicated map PaidCard.fromPressedContent
[error]                                                                         ^
[error] one error found
[error] (common/compile:compileIncremental) Compilation failed

```

Comment (TBonnin) [inline common/app/layout/Sublink.scala]: Explicitly calling the mapping function with the first param would make the compiler happy :D
```
val cards = collection.curatedPlusBackfillDeduplicated.map(PaidCard.fromPressedContent(_))
```

Comment (lps88) [inline common/app/layout/Sublink.scala]: Good idea 😺

**Review by TBonnin — APPROVED** [review]
scala looks good. Cannot really comment on the css though

**Comment by rich-nguyen** [comment]
css is good

**Pull request merged**